_config.yml

```diff
@@ -7,4 +7,4 @@ defaults:
       path: ""
     values:
       layout:  "default"
-      version: 0.24.8
+      version: 0.25.0
```

Comment:
@jacknguyen This is only available in 0.25.1. Please update this line, then I'll accept. Thanks! :ribbon: